eth/tracers/tracers.go

```diff
@@ -26,7 +26,7 @@ import (
 	"github.com/ethereum/go-ethereum/eth/tracers/internal/tracers"
 )
 
-// Tracer interface extends vm.JSTracer and additionally
+// Tracer interface extends vm.Tracer and additionally
```

Comment:
Yeah, there were a lot of accidental renamings, sorry about that. I let my IDE go wild there :)